sparse/file.go

```diff
@@ -33,195 +31,56 @@ func SetupFileIO(direct bool) {
 		fileOpen = fio.OpenFile
```

Comment:
Just tried different ways doing this, but they don't look good either, due to places we need both *os.FIle and those function interfaces. I will skip doing this today for now. Maybe come back to this later discussing with you tomorrow.